lib/KTextbox/__tests__/KTextbox.spec.js

```diff
@@ -212,3 +221,79 @@ describe('KTextbox component', () => {
     });
   });
 });
+
+describe.visual('KTextbox visual test', () => {
+  const snapshotOptions = { widths: [400], minHeight: 512 };
+  it('render KTextbox with label', async () => {
+    await renderComponentForVisualTest('KTextbox', {
+      label: 'Input with label',
+      value: 'Input with label',
+    });
+    await takeSnapshot('KTextbox - label', snapshotOptions);
+  });
+  it('render disabled KTextbox', async () => {
+    await renderComponentForVisualTest('KTextbox', {
+      lable: 'Disabled',
+      value: 'Input with disabled',
+      disabled: true,
+    });
+    await takeSnapshot('KTextbox - disabled', snapshotOptions);
+  });
+  it('render readonly KTextbox', async () => {
+    await renderComponentForVisualTest('KTextbox', {
+      lable: 'Readonly',
+      value: 'Input with readonly',
```

Comment:
```suggestion
      value: 'Readonly input',
```

Minor detail, just sounds a bit more intuitive